content/README.md

```diff
@@ -0,0 +1,124 @@
+# CWL Website 2.0
+
+The in-progress redesign of the CWL Website
+
+The preview build of the site is currently found at [https://deploy-preview-75--cwl-website.netlify.app/](https://deploy-preview-75--cwl-website.netlify.app/)
+
+## To-Do
+
+### Mobile
+
+* Mobile nav should close when you tap outside of it
+* Ensure "Back to Top" button shows up for long pages
+* Users Gallery Submenus should be drop downs
+
+### Broken Links
+
+Users Gallery - Jekyll is inserting a blank link between US National Institutes of Health (NIH) & 
+RAPT. The code renders like this:
+
+```html
+<div class="adopter-entry">
+  <div class="adopter-row-top">
+    <a href="" class="adopter-link"></a>
+  </div>
+</div>
+```
+
+### WebAIM Accessibility
+
+#### Video Player
+
+* The "broken ARIA menu" seems to be an error on the WebAIM tool's part...
+  - It says that it has to have at least one item with `role="menuitem"` - which is satistfied...
+  - Test with screenreader or emulator if possible
```

Comment:
Thanks so much for this feedback!!

> If I can see blurry/large text areas, and use a screen reader, it can be confusing since navigating with tabs moves the title elements under the top menu

Ah right! That's on my to-do list (though I forgot to add it in the README update)

> There appears to be some links to top or back I think, that are announced with the headers of the page. That's distracting since I think I would prefer to hit Home or re-load the page to get Orca to re-read it to me.

To clarify: are you referring to the Back to Top button on the bottom right of the page, or the auto-generated anchor links next to each heading. For example: `<h2 id="vision">Vision of the CWL Project <a href="#vision" class="heading__anchor-symbol">§</a></h2>`. In either case, it sounds like it would be beneficial to hide it from screen readers (since they can already navigate to the headers anyway). I'll add that to my list.

> The GettingStarted page has a text in Japanese, but Orca reads it as "CWL -japanese character- -chinese character- -chinese character-...."

Noted: I'll add the appropriate lang attributes to those

> There were times when I was trying to navigate the menu and I ended up going to the page header, then to the social media buttons, then to donate, and back to About? Not sure why but navigation didn't work as expected sometimes.

Hmmm, ok. I'll look into that.

> But really great job on preparing the ground for good accessibility in the site. Rarely devs worry about ARIA, WCAG, WebAIM, etc, when writing pages. As a potential future user of screen readers (most of my father family had serious eye issues as they got older) I really appreciate it! Thanks!

Thanks so much for the feedback - it's very helpful! It's not my main area of expertise, but I try to do my best. I'll look into screen reader software so I can test it more effectively going forward!